creator/c2m2_submissions/edam_mapper.py

```diff
@@ -0,0 +1,131 @@
+import os
+import csv
+import logging
+import requests
+from dataclasses import dataclass
+from owlready2 import get_ontology
+from typing import Optional, TypedDict, Dict
+
+logger = logging.getLogger(__name__)
+
+EDAM_PATH = "./edam.owl"
+
+Entry = TypedDict("Entry", {"label": str, "description": str})
+
+
+DATA_TYPES = {
+    "Aligned Reads": "Alignment",
+    "Aligned Reads Index": "Genome index",
+    "Annotated Somatic Mutations": None,
+    "Expression": "Expression data",
+    "Gene Expression": "Expression data",
+    "Gene Fusions": None,
+    "gVCF": "Sequence variations",
+    "gVCF Index": None,
+    "Histology Images": "Image",
+    "Isoform Expression": "Expression data",
+    "Operation Reports": "Report",
+    "Other": None,
+    "Pathology Reports": "Report",
+    "Radiology Images": "Report",
+    "Radiology Reports": "Report",
+    "Simple Nucleotide Variations": None,
+    "Somatic Copy Number Variations": "Copy number variation",
+    "Somatic Structural Variations": "Structural variation",
+    "Unaligned Reads": "DNA sequence",
+    "Variant Calls": "VCF",
+    "Variant Calls Index": None,
+}
+
+
+class EDAMMapper:
```

Comment:
What is the purpose of the EDAMMapper? A quick blurb in the docstring would be helpful